archetype/config/models.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Models Configuration
+ *
+ * @see {@link http://trailsjs.io/doc/trailpack/config
+ */
+module.exports = {
+  /**
+   * The default store if not defined by model
+   */
+  defaultStore: 'sqlitedev',
+  /**
+   * The default migration strategy if not defined by the store or model
+   * ENUM: none, safe, drop, alter,
+   */
+  migrate: 'drop'
+
+  /**
+   * You can also define model by model which store, migration, etc it will use.
+   * A la carte model definitions
+   * eg:
+   *
+   * Test: {
```

Comment:
Maybe it's overkill because we're suppose to have this already by putting this configuration under `config` method of each models, but why not ^^ just it's a lot of places for the same feature

Reply:
I understand. However, a lot of times a dev will create a trailpack with a model definition already included. Or need to change the store/tableName/options per environment, and this allows for that.